Makefile

```diff
@@ -3,8 +3,9 @@ MAKEFLAGS+=-r
 
 config=debug
 defines=standard
+cxxstd=default
```

Comment:
> it may distract some people of using pugixml for their projects

I've never heard of anything like that. Having an option to default to no -std usage feels fine.